prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<title>HTML Syntax</title>
```

Comment:
You are missing your `<head>` tag here, the `<title>` tags should be within the `<head>` tags.